### ISSUE TITLE
For Help

### ISSUE THREAD

**Issue by hgbzzw** [yannforget/conda-recipes#1]
Are these packages just for Linux?not for Windows?My anaconda is under Windows,What should i do if I want to use them?

**Comment by yannforget** [comment]
Yes they are for linux. Since Windows is not supported for Ledaps & LaSRC software I think the only way to use them under Windows is to use Cygwin which will provide the required functionnalities (mainly the GCC compiler). It should work in theory but I never tried that.
